UnitsNet.Tests/CustomCode/DensityTests.cs

```diff
@@ -107,25 +107,7 @@ public static void DensityTimesKinematicViscosityEqualsDynamicViscosity()
             DynamicViscosity dynamicViscosity = Density.FromKilogramsPerCubicMeter(2) * KinematicViscosity.FromSquareMetersPerSecond(10);
             Assert.Equal(dynamicViscosity, DynamicViscosity.FromNewtonSecondsPerMeterSquared(20));
         }
-
-        [Fact]
```

Comment:
Same here, shouldn't we keep the Density tests?

Reply:
I think not- I've left the operators as obsolete, tests were "converted"